### PR TITLE
Register login

### PR DESCRIPTION
**Pull request by shabanzadeh**

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,2 +1,6 @@
 MONGO_URL=mongodb://mongo:27017/redilab
 
+
+SECRET_KEY=redilab
```
Comment (syedaffanhamdani) [inline .env:4]: Best practice would be to use environment variables for secret key

File: README.md [hunk 2]
```diff
@@ -49,4 +49,67 @@ on macOS/Linux
 
 ## Run the Application
```
Comment (syedaffanhamdani) [inline README.md:51]: +1

File: config/config.py [hunk 3]
```diff
@@ -0,0 +1,6 @@
+import os
+from dotenv import load_dotenv
+load_dotenv()
+
+SECRET_KEY = os.getenv("SECRET_KEY", "default_secret_key")
```
Comment (syedaffanhamdani) [inline config/config.py]: +1

File: .env.sample [hunk 4]
```diff
@@ -0,0 +1 @@
+MONGO_UR= "mongodb://mongo:27017/redilab"
```
Comment (syedaffanhamdani) [inline .env.sample]: may be should be named to URI or URL

File: routes/user.py [hunk 5]
```diff
@@ -1,22 +1,69 @@
...
     user = users_serializer(collection.find({"_id": _id.inserted_id}))
     return {"status": "Ok","data": user}
 
+
+@user.post("/login")
+async def login_user(user: User):
+    found_user = collection.find_one({"email": user.email})
+    
+    if not found_user:
+        raise HTTPException(status_code=400, detail="User not found")
```
Comment (syedaffanhamdani) [inline routes/user.py]: I would stick to one style only, i.e. status.HTTP_400_BAD_REQUEST rather than 400, though it output is the same.

Comment (shabanzadeh) [reply routes/user.py]: I have done it

File: .env.sample (hunk 4 above)
Comment (shabanzadeh) [reply .env.sample]: I have done it

**Pull request merged**